crates/bevy_ecs/src/system/query.rs

```diff
@@ -381,6 +381,14 @@ pub struct Query<'world, 'state, D: QueryData, F: QueryFilter = ()> {
     this_run: Tick,
 }
 
+impl<D: ReadOnlyQueryData, F: QueryFilter> Clone for Query<'_, '_, D, F> {
+    fn clone(&self) -> Self {
+        *self
```

Comment:
sorry, noob question: how does this work?
The `&QueryState` inside the Query is simply copied? i.e. we create a new Query that points to the same QueryState?